Expecto/Expecto.fs

```diff
@@ -643,3 +643,44 @@ module Tests =
   /// Returns 0 if all tests passed, otherwise 1
   let runTestsInAssemblyWithCLIArgs cliArgs args =
     runTestsInAssemblyWithCLIArgsAndCancel CancellationToken.None cliArgs args
+
+  let runTestsReturnLogs cliArgs args tests =
```

Comment:
I could use feedback on this method name.

Reply:
Maybe `runTestsForInteractive`

Reply:
What do you think about `runTestsInteractively`?

Reply:
I thought of a similar name, but I feel like it denotes the test run itself is interactive, as in there will be decision points for the user during the test run.

Reply:
Well, `runTestsForInteractive` seems reasonable to me, then.